inbc-program/tests/unit/test_source_app_parser.py

```diff
@@ -30,11 +30,11 @@ def test_parse_add_arguments_deb822_format_separate_lines_successfully(self):
         f = self.arg_parser.parse_args(
             ['source', 'application', 'add',
              '--sources', 'X-Repolib-Name: Google Chrome',
-                          'Enabled: yes',
-                          'Types: deb',
-                          'URIs: https://dl-ssl.google.com/linux/linux_signing_key.pub',
-                          'Suites: stable',
-                          'Components: main',
+             'Enabled: yes',
```

Comment:
Verify that the indentation changes in the test cases do not affect the parsing of arguments, as alignment can sometimes be significant in certain formats. [medium]